css-brochure-form/styles.css

```diff
@@ -3,7 +3,6 @@
 }
 
 body {
-  width: 65%;
   margin: auto;
 }
```

Comment:
You can probably just remove this rule entirely. With no width set on `body`, the `margin: auto` will have no effect anyway 🤓